project/dimensions/enduses.csv

```diff
@@ -0,0 +1,99 @@
+natural_gas_cooking_range,Cooking Range,natural_gas,MBtu
+natural_gas_fireplace,Fireplace,natural_gas,MBtu
+natural_gas_grill,Grill,natural_gas,MBtu
+natural_gas_heating,Heating,natural_gas,MBtu
+natural_gas_hot_tub_heater,Hot Tub Heater,natural_gas,MBtu
+natural_gas_interior_equipment,Interior Equipment,natural_gas,MBtu
+natural_gas_lighting,Lighting,natural_gas,MBtu
+natural_gas_pool_heater,Pool Heater,natural_gas,MBtu
+natural_gas_water_systems,Water Systems,natural_gas,MBtu
+propane_clothes_dryer,Clothes Dryer,propane,MBtu
+propane_cooking_range,Cooking Range,propane,MBtu
+propane_heating,Heating,propane,MBtu
+propane_water_systems,Water Systems,propane,MBtu
+wood_heating,Heating,wood,MBtu
+electricity_ev_dcfc,EV DCFC,electricity,MWh
+electricity_ev_l1l2,EV L1/L2,electricity,MWh
+electricity_conventional_boiler,Conventional Boiler,electricity,MWh
+fuel_oil_conventional_boiler,Conventional Boiler,fuel_oil,MBtu
+natural_gas_conventional_boiler,Conventional Boiler,natural_gas,MBtu
+hgl_conventional_boiler,Conventional Boiler,hgl,MBtu
+coal_conventional_boiler,Conventional Boiler,coal,MBtu
+electricity_process_heating,Process Heating,electricity,MWh
+fuel_oil_process_heating,Process Heating,fuel_oil,MBtu
+natural_gas_process_heating,Process Heating,natural_gas,MBtu
+hgl_process_heating,Process Heating,hgl,MBtu
+coal_process_heating,Process Heating,coal,MBtu
+electricity_process_cooling_and_refrigeration,Process Cooling and Refrigeration,electricity,MWh
+natural_gas_process_cooling_and_refrigeration,Process Cooling and Refrigeration,natural_gas,MBtu
+electricity_machine_drive,Machine Drive,electricty,MWh
+fuel_oil_machine_drive,Machine Drive,fuel_oil,MBtu
+natural_gas_machine_drive,Machine Drive,natural_gas,MBtu
+hgl_machine_drive,Machine Drive,hgl,MBtu
+coal_machine_drive,Machine Drive,coal,MBtu
+electricity_electro-chemical_processes,Electro-Chemical Process,electricity,MWh
+electricity_other_process,Other Process,electricity,MWh
+fuel_oil_other_process,Other Process,fuel_oil,MBtu
+natural_gas_other_process,Other Process,natural_gas,MBtu
+hgl_other_process,Other Process,hgl,MBtu
+coal_other_process,Other Process,coal,MBtu
+electricity_facility_hvac,Facility HVAC,electricity,MWh
+natural_gas_facility_hvac,Facility HVAC,natural_gas,MBtu
+electricity_facility_lighting,Facility Lighting,electricity,MWh
+electricity_other_facility_support,Other Facility Support,electricity,MWh
+natural_gas_other_facility_support,Other Facility Support,natural_gas,MBtu
+electricity_onsite_transportation,Onsite Transportation,electricity,MWh,
+fuel_oil_onsite_transportation,Onsite Transportation,fuel_oil,MBtu
+natural_gas_onsite_transportation,Onsite Transportation,natural_gas,MBtu
+hgl_onsite_transportation,Onsite Transportation,hgl,MBtu
+electricity_other_nonprocess,Other Nonprocess,electricity,MWh
+fuel_oil_other_nonprocess,Other Nonprocess,fuel_oil,MBtu
+natural_gas_other_nonprocess,Other Nonprocess,natural_gas,MBtu
```

Comment:
I pulled these from MECS 2018: https://www.eia.gov/consumption/manufacturing/data/2018/pdf/Table5_5.pdf. Are they accurate? Which fuels does industry plan to include?